### ISSUE TITLE
Task Delete and Create new - Keeps the TaskNo, WorkDone, Uploaded Files

### ISSUE THREAD

**Issue by adarshmadrecha** [adarshmadrecha/jamku#297]
Suggested by `Vatsal Bhatia`

## Feature Request
Because task creation is decentralized, there are sometimes scenarios where Client or Service needs to be changed.
Currently the option is to delete the Task and again create the same.
But, since there can be workdone and files already added it's difficult to delete the same.

Hence, this feature request, where there is an option to delete existing task which keeps the following things intact
1. Task No
2. Workdone
3. Documents (Inward Outward Register)
4. Sub Task
5. Task Notes
6. Uploaded Files
7. Todos
8. Stages ** 
9. Custom Fields  ** 

** - Only if there is **no** change in Service.

## Implementation Impact
In short, this feature will allow us to change following for a task
1. Client 
2. Service

**Comment by vatsalbhatia** [comment]
In case of a service is changed from one to another..
Custom Fields and Stages would be deleted....
New Task Created in place of it may have written description of the deleted info into Task Description....
At the time accepting to create new task in place of older one, it may ask whether you wish to save details in Task Desceiption....
